web/src/components/common/UI/EditSubmitButton.tsx

```diff
@@ -14,27 +14,31 @@
 // limitations under the License.
 //
 
-import React from 'react';
 import { Label } from '@rebass/forms';
+import React from 'react';
 import { StyledFormButton, StyledFormDisabledButton } from './Button';
 
-export const EditSubmitButton = ({ hasPendingEdit, isProvisioned, pristine }: any) => {
+export const EditSubmitButton = (props: any) => {
+  const { hasPendingEdit, isProvisioned, pristine } = props;
+
   if (pristine) {
     return (
       <>
-        {/* @ts-ignore */}
         <StyledFormDisabledButton disabled style={{ display: 'block' }}>
           Request Update
         </StyledFormDisabledButton>
       </>
     );
   }
   if (!hasPendingEdit && isProvisioned) {
-    return <StyledFormButton style={{ display: 'block' }}>Request Update</StyledFormButton>;
+    return (
+      <>
+        <StyledFormButton style={{ display: 'block' }}>Request Update</StyledFormButton>
+      </>
+    );
   }
   return (
     <>
-      {/* @ts-ignore */}
       <StyledFormDisabledButton disabled style={{ display: 'block' }}>
```

Comment:
no need to change but it is very interesting to change a button style from default `inline-block` to `block`. I'd personally just increase the width